package.json

```diff
@@ -1,11 +1,11 @@
 {
     "name": "better-cpp-syntax",
-    "displayName": "Better C++ Syntax",
-    "description": "The bleeding edge of the VS Code C++ syntax",
+    "displayName": "Better Syntax",
+    "description": "The bleeding edge of some VS Code syntaxes",
```

Comment:
maybe list the languages or say C-like. "some VS Code syntaxes" is pretty vague.